RSignatureVerifier/APIClient.swift

```diff
@@ -0,0 +1,71 @@
+protocol SessionProtocol {
+    func startTask(with request: URLRequest, completionHandler: @escaping (Data?, URLResponse?, Error?) -> Void)
+}
+
+extension URLSession: SessionProtocol {
+    func startTask(with request: URLRequest, completionHandler: @escaping (Data?, URLResponse?, Error?) -> Void) {
+        dataTask(with: request) { (data, response, error) in
+            completionHandler(data, response, error)
+        }.resume()
+    }
+}
+
+internal struct Response {
```

Comment:
I think that it is internal by default, you can remove it.

Reply:
You can leave it as internal for now